apps/examples/stdk_smart_switch/device_info.json

```diff
@@ -0,0 +1,8 @@
+{
+	"deviceInfo": {
+		"firmwareVersion": "firmwareVersion_here",
+		"privateKey": "privateKey_here",
+		"publicKey": "publicKey_here",
+		"serialNumber": "serialNumber_here"
+	}
+}
```

Comment:
could you add space?

Reply:
this config file need to be overwrited when enabel this example, is it mandatory to add space here?

Reply:
Yes, It's coding rule.